### DISCUSSION TITLE
Select multiple not working outside panel

### DISCUSSION THREAD

**Discussion by gpinyol101** [filamentphp/filament#11922] [Help]
### Package

Form builder

### Package Version

v3.0.0.

### How can we help you?

My problem is that ->multiple() on Select Form Component is working correctly if I use it on panel, but I use panel and also public page.
So I'm creating a new page with Livewire for a public users and ->multiple() is not working there. Maybe is this because on panel there is some alpine library already loaded and on public pages is not?

This is what I use:
Forms\Components\Select::make('exclusions')->multiple()->options(['test' => 'Test'])

Not sure what is missing on public pages that is not by default on panel filament

Any idea or suggestion will be grateful!

Thanks a lot

**Comment by leandrocfe** [comment]
Did you follow these steps? https://filamentphp.com/docs/3.x/forms/adding-a-form-to-a-livewire-component#adding-the-form

You should initialize the form with `$this->form->fill()` in `mount()` method and configure the `statePath`

**Reply by gpinyol101** [reply]
Yes, I had everything set up, and the fill is working correctly and all the others fields too. The only problem is the multiple() that makes the select completly broken.
Same form on panel works and on public page doesn't. I think some library is missing from alpine o filament when is visiting public pages.
I searched a lot and I spent a lot of hours with that and I can't find a solution.

**Reply by leandrocfe** [reply]
Console errors?
Share the whole code you are using please.

**Reply by gpinyol101** [reply]
Don't know if this video will help you, or maybe you need more information.
https://www.loom.com/share/f1a826030262499f89268bb453a0d3b4?sid=c1b6776e-9477-4b34-9e98-7a9807838d6c

First of all I show you that the same select it works correctly on panel filament, and then I show you that in public pages, on livewire component, it doesn't work.

Thank you!

**Reply by gpinyol101** [reply]
Hi! any solution??
Thank you!
